IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "_delete": [
+    {
+      "classKind": "Participant",
+      "iid": "284334dd-e8e5-42d6-bc8a-715c507a7f02",
+      "domain": [ "0e92edde-fdff-41db-9b1d-f2e484f12535" ]
+    }
+  ],
+  "_create": [],
+  "_update": [
+    {
+      "classKind": "Participant",
+      "iid": "284334dd-e8e5-42d6-bc8a-715c507a7f02",
+      "modifiedOn": "2020-04-14T17:42:59.264Z",
+      "selectedDomain": "00000000-0000-0000-0000-000000000000"
```

Comment:
The value here `"00000000-0000-0000-0000-000000000000"` is not an empty value. 
It should be `null`

Reply:
We do not use Nullable<Guid> so, Guid is Not Nullable always comes as empty.